List/index.d.ts

```diff
@@ -4,17 +4,19 @@ import {VNode} from 'preact';
 declare interface IListProps extends JSX.HTMLAttributes {
   dense?: boolean;
   'two-line'?: boolean;
-  interactive?: boolean;
+  avatar?: boolean;
```

Comment:
Is this supposed to be `avatar` or `avatar-list`, because in `_mdcProps` it says `avatar-list`?

Reply:
Done